processor/file.go

```diff
@@ -183,8 +185,11 @@ func walkDirectoryParallel(root string, output chan *FileJob) {
 						totalCount++
 						mutex.Unlock()
 
-						LoadLanguageFeature(language)
-						output <- &FileJob{Location: fpath, Filename: f.Name(), Extension: extension, Language: language}
+						for _, l := range language {
+							LoadLanguageFeature(l)
+						}
+
+						output <- &FileJob{Location: fpath, Filename: f.Name(), Extension: extension, Language: language[0], Languages: language}
```

Comment:
If this code is no longer responsible for determining the "real" language, I'd argue for not setting the `Language` field here (or on line 265). I'd keep the responsibility in one place, probably in `workers.go` with the condition to guess languages.

Reply:
That's a fair point. I initially kept it the same to avoid further processing down the line as it knows the language at this point.

Ill have a look if there is any performance impact from this (though I suspect there should be none).